fipy/meshes/abstractMesh.py

```diff
@@ -692,6 +692,80 @@ def _localOverlappingFaceIDs(self):
         """
         return self.topology._localOverlappingFaceIDs
 
+    @property
+    def _vertexCellIDs(self):
```

Comment:
This isn't actually used anywhere. Do you see a likely use for them?

Reply:
very nice docs BTW

Reply:
I needed `_vertexFaceIDs` for my parallel Gmsh writer and remembered that you'd done most of the work on StackOverflow. Once I implemented it, I realized I could radically simplify things but just using `_vertexCellIDs`, but figured I'd keep them both.

I'm making this a separate pull request just to keep things as atomic as possible. Gmsh I/O is going to be a monster code review as it is.

Reply:
Oh, I just thought you were doing this for fun :-)